openedx/features/edly/cookies.py

```diff
@@ -0,0 +1,69 @@
+from django.conf import settings
+
+from openedx.core.djangoapps.user_authn.cookies import standard_cookie_settings
+from openedx.features.edly.models import EdlySubOrganization
+from openedx.features.edly.utils import encode_edly_user_info_cookie
+
+
+def set_logged_in_edly_cookie(request, response, user):
+    """
+    Set edly user info cookie at the time of user login.
+
+    Arguments:
+        request (HttpRequest): The request to the view, used to calculate
+            the cookie's expiration date based on the session expiration date.
+        response (HttpResponse): The response on which the cookie will be set.
+        user (User): The currently logged in user.
+
+    Returns:
+        HttpResponse
+
+    """
+    if user.is_authenticated and not user.is_anonymous:
+        cookie_settings = standard_cookie_settings(request)
+        edly_cookie_string = _get_edly_cookie_string(request)
+
+        response.set_cookie(
+            settings.EDLY_USER_INFO_COOKIE_NAME.encode('utf-8'),
```

Comment:
No need for `utf-8` encoding we have hardcoded value for edly user info cookie name with only alphanumeric values

Reply:
done. this was done in the original code and there were hardcoded values there as well so I did this for consistency.